packages/assets-controllers/src/NftDetectionController.ts

```diff
@@ -122,7 +122,7 @@ export interface ApiNftCreator {
  */
 export interface NftDetectionConfig extends BaseConfig {
   interval: number;
-  chainId: `0x${string}` | `${number}` | number;
+  chainId: string;
```

Comment:
This is only ever set via the network controller provider config, so it has been updated to be just `string`, which is what the type is at the moment in the network controller. This will get more specific in an upcoming PR.